Say Their Names/Source/Controller/Base/ServiceClientViewController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ServiceClientViewController.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class ServiceClientViewController : UIViewController, ServiceClient  {
```

Comment:
We already have `BaseViewController`. I'm personally not in favour of base classes unless absolutely necessary, but even if we do have them, let's have one, not two :)

Reply:
I completely agree with @hybridcattt, we must avoid as much we can to have "bases".

Reply:
Okay, done.  I now just conform HomeViewController to ServiceReferring.  

I was just trying to follow the conventions of the project as I saw it.  My plan was simply to gradually move all view controllers over to ServiceClientViewController subclasses as we switched from xib to programmatic UI (since BaseViewController as written won't let us init without using a nib). 

If the consensus is that we don't need a base view controller, I'm very happy with that.

Reply:
That sounds great! I don’t look too closely at the BaseViewController. If you see any inconsistencies there with regards to programmatic UI, feel free to fix it or create a Trello card 😊